src/nxt_cgroup.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#ifndef _NXT_CGROUP_H_INCLUDED_
+#define _NXT_CGROUP_H_INCLUDED_
+
+#include <nxt_main.h>
+#include <nxt_process.h>
+
+typedef struct {
```

Comment:
For the change I'm planning to be more iwyu(1) friendly, I'll probably need to separate typedefs from structure definitions.  Maybe it would be good to start doing it already:

```c
typedef struct nxt_cgroup_s  nxt_cgroup_t;


struct nxt_cgroup_s {
    ...
}
```

Also, I'd put the structure definition after all typedefs.

Reply:
OK, good idea.

Should there be one or blank lines between the last typedef and the structure definition?

Reply:
2 seems to be more consistent with the guidelines.

Reply:
2 it is then. You can tell I never double space anything!